it/controllers/APIControllerISpec.scala

```diff
@@ -435,6 +439,7 @@ class APIControllerISpec extends IntegrationSpecCommonBase with ETMPWiremock wit
             |       "principalChargeMainTransaction" : "4700",
             |       "penaltyAmountOutstanding": 99.99,
             |       "penaltyAmountPaid": 1001.45,
+            |       "penaltyAmountPosted": 1101.44,
```

Comment:
This would be 0

Reply:
It's an accruing penalty :)

Reply:
Ok, I'll change paid, posted and outstanding to 0